code/game/objects/structures/barsign.dm

```diff
@@ -383,21 +738,46 @@
 	icon = "themaint"
 	desc = "Home to Greytiders, Security and other unholy creations."
 
+/datum/barsign/syndicat
+	name = "The SyndiCat"
+	icon = "thesyndicat"
+	desc = "Syndicate or die."
+	syndicate = TRUE
+
 /datum/barsign/hiddensigns
-	hidden = 1
+	hidden = TRUE
 
 //Hidden signs list below this point
 /datum/barsign/hiddensigns/empbarsign
 	name = "Haywire Barsign"
 	icon = "empbarsign"
 	desc = "Something has gone very wrong."
 
-/datum/barsign/hiddensigns/syndibarsign
-	name = "Syndi Cat Takeover"
-	icon = "syndibarsign"
-	desc = "Syndicate or die."
-
 /datum/barsign/hiddensigns/signoff
 	name = "Bar Sign"
-	icon = "empty"
+	icon = "off"
 	desc = "This sign doesn't seem to be on."
+
+/datum/barsign/hiddensigns/signbroken
+	name = "Broken Bar Sign"
+	icon = "broken"
+	desc = "This sign has a massive crack in it!"
```

Comment:
```suggestion
	name = "Broken Bar Sign"
	desc = "This sign has a massive crack in it!"
	icon = "broken"
```

Reply:
Sure?  That's the format for all of them so I've changed it on every sign.